mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx

```diff
@@ -462,17 +462,13 @@ export function CompositeSwapScreen ({ route }: Props): JSX.Element {
             <View style={tailwind('mx-4')}>
```

Comment:
```suggestion
            <View style={tailwind('mx-4 flex-1')}>
```
This fixes the vertical spacing known issue

Reply:
omg, thanks @pierregee !!! ❤️